package.json

```diff
@@ -7,6 +7,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.12.1",
     "@fortawesome/react-fontawesome": "^0.1.9",
     "@material-ui/core": "^4.9.5",
+    "@material-ui/icons": "^4.9.1",
```

Comment:
we definatly don't need this, see how we are using icons in other places

Reply:
Then toggling of icons is not happening while using font awesome icons.. I don't know why?

Reply:
Then the toggling of the icons is not happening while using <i>. I don't know why?

Reply:
`RenderWishlistButton = () => {
		const { isWishlisted } = this.state;
        if(isWishlisted){
            return (    
                <span className="icon" key={'filled'}>
                    <BootstrapTooltip title={"Added to Wishlist"}>
                        <IconButton disableRipple onClick={this.handleWishlistChange}>
						<i className={"fas fa-heart" }></i> 
                        </IconButton>
                    </BootstrapTooltip>
                </span>
            );
        } else {
			return (
				<span className="icon" key={'bordered'}>
                    <BootstrapTooltip title={"Added to Wishlist"}>
                        <IconButton disableRipple onClick={this.handleWishlistChange}>
							<i className={"far fa-heart" }></i> 
                        </IconButton>
                    </BootstrapTooltip>
                </span>
			)
		}
    };`